### DISCUSSION TITLE
is this package working on web platform?

### DISCUSSION THREAD

**Discussion by Trung15010802** [dsrenesanse/worker_manager#98] [Q&A] [answered]
If it is. Could you make an example how to parse json on web please?

**Comment by dsrenesanse** [comment, marked as answer]
> If it is. Could you make an example how to parse json on web please?

hi, worker_manager works same as compute from foundation, it is just a future.

**Comment by Lucaffo** [comment]
Can't talk about my use case since I tried to use it as a worker library but I can't get it to work properly so I gave up and use the default worker code in dart:html The library seems really good tbh! Great work!

What I mean but standard approach is to create a new dart file and compile it by itself then with the worker class referring to the worker by the .js file name. 

https://api.dart.dev/stable/2.1.1/dart-html/Worker-class.html

Btw, my problem was that the worker lambda code stopped the main thread.
